libethash-cuda/ethash_cuda_miner_kernel.cu

```diff
@@ -26,7 +26,7 @@
 #endif
 
 __global__ void 
-__launch_bounds__(TPB, BPSM)
+//__launch_bounds__(TPB, BPSM)
```

Comment:
Sure, I can.
TPB and BPSM are defined in the same file (libethash-cuda/ethash_cuda_miner_kernel.cu) with launch_bound